<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>577</width>
    <height>590</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>580</width>
    <height>590</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>370</y>
      <width>281</width>
      <height>191</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../Downloads/download.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>281</width>
      <height>171</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../Downloads/download.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>281</width>
      <height>171</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../Downloads/download.png</pixmap>
    </property>
   </widget>
   <widget class="QProgressBar" name="intensityBar">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>290</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="maximum">
     <number>10</number>
    </property>
    <property name="value">
     <number>8</number>
    </property>
    <property name="format">
     <string>%v</string>
    </property>
   </widget>
   <widget class="QPushButton" name="timeButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>480</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Time</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../Downloads/smallclock.png</normaloff>../Downloads/smallclock.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="downButton">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>480</y>
      <width>21</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>v</string>
    </property>
   </widget>
   <widget class="QPushButton" name="upButton">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>480</y>
      <width>21</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>^</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>480</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../Downloads/smallLock.png</normaloff>../Downloads/smallLock.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="powerButton">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>520</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Power</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>200</y>
      <width>171</width>
      <height>81</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>140</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../Downloads/goodSmallClock.png</pixmap>
    </property>
   </widget>
   <widget class="QProgressBar" name="batteryBar">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>41</width>
      <height>71</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <property name="format">
     <string/>
    </property>
   </widget>
   <widget class="QListWidget" name="treatmentListWidget">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>330</y>
      <width>181</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="historyButton">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>440</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>History</string>
    </property>
   </widget>
   <widget class="QPushButton" name="intensityButton">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>440</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Intensity</string>
    </property>
   </widget>
   <widget class="QLabel" name="batteryLabel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>110</y>
      <width>41</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>text</string>
    </property>
   </widget>
   <widget class="QWidget" name="frequencyWidget" native="true">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>81</width>
      <height>80</height>
     </rect>
    </property>
    <widget class="QRadioButton" name="freq77hz">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>61</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>77 Hz</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="freq100hz">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>60</y>
       <width>61</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>100 Hz</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="freq05hz">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>61</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>0.5 Hz</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>71</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Noto Sans Adlam</family>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Frequency</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="waveformWidget" native="true">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>20</y>
      <width>81</width>
      <height>80</height>
     </rect>
    </property>
    <widget class="QRadioButton" name="waveBeta">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>61</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Beta</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>71</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Noto Sans Adlam</family>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Wave form</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="waveAlpha">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>61</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Alpha</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="waveGamma">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>60</y>
       <width>71</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Gamma</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="timerGroup">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>130</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Noto Sans Adlam</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string> Countdown Cycles</string>
    </property>
    <widget class="QRadioButton" name="timer40Radio">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>30</y>
       <width>41</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Noto Sans</family>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>40</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="timer20Radio">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>41</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Noto Sans</family>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>20</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="timer60Radio">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>30</y>
       <width>41</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Noto Sans</family>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>60</string>
     </property>
    </widget>
   </widget>
   <widget class="QCheckBox" name="recordCheckBox">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>410</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Record at the end of treatment</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>20</y>
      <width>241</width>
      <height>281</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Noto Sans Adlam</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>Device Internal Controller</string>
    </property>
    <widget class="QComboBox" name="inContactDropdown">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>30</y>
       <width>91</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Noto Sans</family>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <item>
      <property name="text">
       <string>True</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>False</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="chargeDeviceButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>221</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Noto Sans</family>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Charge Device to 100%</string>
     </property>
    </widget>
    <widget class="QPushButton" name="exceedCurrentButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>221</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Noto Sans</family>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Exceed Current Limit</string>
     </property>
    </widget>
    <widget class="QPushButton" name="fastForwardButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>221</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Noto Sans</family>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Fast-forward timer to 2s remaining</string>
     </property>
    </widget>
    <widget class="QWidget" name="batteryHandleWidget" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>221</width>
       <height>41</height>
      </rect>
     </property>
     <widget class="QPushButton" name="drainBattery1">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>10</y>
        <width>61</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Noto Sans</family>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>-1%</string>
      </property>
     </widget>
     <widget class="QPushButton" name="drainBattery10">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>10</y>
        <width>61</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Noto Sans</family>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>-10%</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Noto Sans</family>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Drain battery:</string>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>151</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Noto Sans</family>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Attach Probe To Skin:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="fixCurrentButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>230</y>
       <width>221</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Fix Exceed Current Limit</string>
     </property>
    </widget>
   </widget>
   <widget class="QCheckBox" name="poweredBox">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>140</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>Circuit Check</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>577</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuCOMP_3004_TEAM_23">
    <property name="title">
     <string>COMP 3004 - TEAM 21: Zachary Hamel + David Chen + Minh Thang Cao</string>
    </property>
   </widget>
   <addaction name="menuCOMP_3004_TEAM_23"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
